src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
I did. The interactive part is in install.c now. It uses the -l to get a list from loaddosinstall and uses -d to download a dos to a temporary directory.